pyproject.toml

```diff
@@ -138,6 +137,7 @@ test_extra = [
     "imageio-ffmpeg>=0.4.1",
     "snirf",
     "neo",
+    "mne_bids",
```

Comment:
```suggestion
    "mne-bids",
```
no?

Reply:
on pypi yes: https://pypi.org/project/mne-bids/ ... but don't hyphens and dashes get "normalized" anyhow (so it doesn't matter)?

Reply:
yes it doesn't matter, it's a question of aesthetics -- and the package is called mne-bids, not mne_bids (this is just the way it's imported)